scripts.py

```diff
@@ -1309,6 +1309,14 @@ def arg_parser(cls, _db):
             help=u"Show what would be imported, but don't actually do the import.",
             action='store_true',
         )
+        parser.add_argument(
+            '--default-medium-type',
+            help=u'Default medium type used in the case when it\'s not explicitly specified in a metadata file. '
+                 u'Valid values are: Book, Audio.',
```

Comment:
Substitute EditionConstants.FULFILLABLE_MEDIA both here and in `choices` below.

Reply:
I changed the code accordingly. You also wanted to replace `u'Valid values are: Book, Audio.'` with `u'Valid values are: EditionConstants.FULFILLABLE_MEDIA.'`, right?

Reply:
Sorry, I should have been more clear. I meant something like this:
```
"Valid values are: {}".format(", ".join(EditionConstants.FULFILLABLE_MEDIA))
```

Reply:
My bad, fixed it